### PR TITLE
Homework 4: Testing

### PR DESCRIPTION
**Pull request by ivan-kuhta**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "@types/express": "^4.17.20",
+    "@types/jest": "^29.5.8",
+    "@types/mongoose": "^5.11.97",
+    "@types/node": "^20.8.9",
+    "@types/supertest": "^2.0.16",
+    "@types/swagger-jsdoc": "^6.0.2",
+    "@types/swagger-ui-express": "^4.1.5",
+    "jest": "^29.7.0",
+    "mongodb": "^6.2.0",
+    "mongoose": "^8.0.0",
```
Comment (IhorOsadchyi) [inline package.json]: Are you sure that it's a devDependencies?

File: src/mongodb/index.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const uri = "mongodb+srv://ivan_kuxta:ivan_kuxta@learnnodejs.nlxd0u0.mongodb.net/?retryWrites=true&w=majority";
```
Comment (IhorOsadchyi) [inline src/mongodb/index.ts]: Please, use .env file for sensitive data like this.

**Review by IhorOsadchyi — CHANGES_REQUESTED** [review]

File: src/mongodb/index.ts (hunk 2 above)
Comment (ivan-kuhta) [reply src/mongodb/index.ts]: thanks for the feedback, I changed the previous pull, but didn't change this one

File: .env
Comment (IhorOsadchyi) [inline .env:1]: You do not need to commit this file to repository. The main idea - not to do it and save sensitive data locally!

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import express, { type Express } from 'express'
+
+import router from './routes'
+import swagger from './swagger'
+import { connection } from './mongodb'
+
+const app: Express = express()
+
+const hostname: string = '127.0.0.1'
+const port: number = 3000
```
Comment (IhorOsadchyi) [inline src/index.ts]: This const also can be moved to .env file or at least to constants.

**Review by IhorOsadchyi — APPROVED** [review]
Please pay attention to the comments!

File: .env
Comment (ivan-kuhta) [reply .env:1]: Yes, this file does not need to be added, but it is done for example.

**Pull request merged**